# Grid map message; needed to construct a GridMap boject
time stamp

int32 width
int32 length
string name

# parameters used to map grid map back to metric space
float32 metric_gx_min
float32 metric_gx_max
float32 metric_gy_min
float32 metric_gy_max
float32 grid_size

sloop_object_search/GridMapLoc[] locations
